### ISSUE TITLE
[CI/CD]: Review coverage of accessibility checks in all-claims end-to-end tests

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#7319]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Applications **must** have thorough end-to-end tests that run in our continuous integration/continuous deployment (CI/CD) pipeline. These tests should include thorough axe checks. While auditing the `/disability-benefits/all-claims` app folder, I didn't see any axe checks in the Puppeteer end-to-end tests. I'd like the front-end engineering team to review this application, and add end-to-end tests as needed. Definition of done in acceptance criteria below.

## Point of Contact

**VFS Point of Contact:** _Jennifer_

## Environment
* `vets-website/src/applications/disability-benefits/all-claims/tests`
* `$ yarn test:e2e src/applications/disability-benefits/all-claims/tests`

## Acceptance Criteria

**Definition of done:**
- [ ] Front-end team member(s) have reviewed end-to-end tests for axe checks
- [ ] axe checks are added if this application is currently running in production
- [ ] axe checks are run for hidden content like modal windows, accordions
- [ ] FE team has consulted with accessibility specialist in cases where there are high numbers of modals, accordions, other hidden content that could slow down e2e test runs.
- [ ] No axe `violations` appear in the plugin console. These will break the CI/CD build.

## WCAG or Vendor Guidance (optional)

* [Custom axeCheck helper method](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/testing/e2e/nightwatch-commands/axeCheck.js)
* [VSP guidance on writing end-to-end tests](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/quality-assurance/e2e-testing)

**Comment by Mottie** [comment]
Related to:
- #8890 - Original Claims a11y spot check
- #8595 - Fix a11y errors in 526

**Comment by 1Copenut** [comment]
@Mottie Were you able to get the axe checks passing in your Puppeteer tests?

**Comment by Mottie** [comment]
Sadly, no. The tests keep timing out and I could not find a solution. I think my next step is to look at switching the e2e tests over to Cypress since that work appears to already have axe checks built in.

**Comment by Mottie** [comment]
I started work on adding axe checks in this PR - https://github.com/department-of-veterans-affairs/vets-website/pull/12241 - but due to an time out error that I still haven't found a solution for, I'll put that work on the puppeteer e2e test on hold for now.

There is a newer Cypress e2e test PR which hopefully will be merged soon - https://github.com/department-of-veterans-affairs/vets-website/pull/12742

**Comment by Mottie** [comment]
Cypress testing was just merged in yesterday (6/15). I will begin work on porting the puppeteer tests over to Cypress if I have time in this next sprint.

**Comment by Mottie** [comment]
There is a PR for all-claims e2e tests here - https://github.com/department-of-veterans-affairs/vets-website/pull/12978; luckily it was already done! I'll see what needs to be done to get it merged.
